pkg/reconciler/v1alpha1/revision/revision.go

```diff
@@ -408,7 +408,7 @@ func (c *Reconciler) reconcile(ctx context.Context, rev *v1alpha1.Revision) erro
 
 		for _, phase := range phases {
 			if err := phase.f(ctx, rev); err != nil {
-				logger.Errorf("Failed to reconcile %s: %v", phase.name, zap.Error(err))
+				logger.Errorf("Failed to reconcile %s: %v", phase.name, err)
```

Comment:
I would rather see this go the other way so the phase name is still in there but Error is searchable in the logs. (Or use sprintf for the phase name).

```suggestion
				logger.Errorw("Failed to reconcile", zap.String("phase", phase.name), zap.Error(err))
```